sqld/src/replication/primary/frame_stream.rs

```diff
@@ -46,6 +53,13 @@ impl FrameStream {
         if matches!(self.state, FrameStreamState::Closed) {
             return;
         }
+        if let Some(max_frames) = self.max_frames {
+            if self.produced_frames == max_frames {
+                tracing::trace!("Max number of frames reached ({max_frames}), closing stream");
```

Comment:
```suggestion
                tracing::debug!("Max number of frames reached ({max_frames}), closing stream");
```

This feels like a debug message tbh